### ISSUE TITLE
Grand Kingdom - CUSA04630

### ISSUE THREAD

**Issue by MrSn0wy** [red-prig/fpps4-game-compatibility#1694]
### Game Name

Grand Kingdom

### Game code

CUSA04630

### Game version

v1.00

### Used emulator's version

9af0047

### Current status

Nothing

### Error

nop nid:libSceSystemGesture:8F887CD8241610DA:sceSystemGestureUpdateTouchRecognizer

### Description

Crashes immediately after launch

### Screenshots

![image](https://github.com/red-prig/fpps4-game-compatibility/assets/61592704/86cf659e-22cc-433f-acfa-2bfb12d5802c)

### Log File

[Log.txt](https://github.com/red-prig/fpps4-game-compatibility/files/14084611/log.txt)

**Comment by gandalfthewhite19890404** [comment]
Add libSceNgs2.prx, retest and edit first post (or delete and create new comment, not issue)

**Comment by MrSn0wy** [comment]
ah alright i will, one moment

**Comment by MrSn0wy** [comment]
After adding the following whitelisted lib:

> libSceNgs2.sprx

it crashes immediately with the error: 
`nop nid:libSceSystemGesture:DE9700BE6C0A0AF3:sceSystemGestureInitializePrimitiveTouchRecognizer`

![image](https://github.com/red-prig/fpps4-game-compatibility/assets/61592704/33bb39e2-5f26-4427-b281-2d7f4d7aa5f9)

**Comment by GamingDucking** [comment]
Retest

**Comment by MrSn0wy** [comment]
It now crashes with: **nop nid:libSceSystemGesture:8F887CD8241610DA:sceSystemGestureUpdateTouchRecognizer**

![Screenshot 2024-01-29 140412](https://github.com/red-prig/fpps4-game-compatibility/assets/61592704/86cf659e-22cc-433f-acfa-2bfb12d5802c)
---
[log.txt](https://github.com/red-prig/fpps4-game-compatibility/files/14084611/log.txt)

**Comment by GamingDucking** [comment]
> It now crashes with: **nop nid:libSceSystemGesture:8F887CD8241610DA:sceSystemGestureUpdateTouchRecognizer**
> 
> ## ![Screenshot 2024-01-29 140412](https://private-user-images.githubusercontent.com/61592704/300467486-86cf659e-22cc-433f-acfa-2bfb12d5802c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY1MzczODEsIm5iZiI6MTcwNjUzNzA4MSwicGF0aCI6Ii82MTU5MjcwNC8zMDA0Njc0ODYtODZjZjY1OWUtMjJjYy00MzNmLWFjZmEtMmJmYjEyZDU4MDJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTI5VDE0MDQ0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg4NWVmNjZlMDQ4NGUwNzYxMDcwMGIyNzdhYjk0YjQ2NGI2YjMwNjIyNDUxNjU2YTAxMTU3ODFmMzdhZjBjYjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.FiEh4zpOffQuutPAwzLtEolaLZ9B_yAQMVQCJvthlYw)
> [log.txt](https://github.com/red-prig/fpps4-game-compatibility/files/14084611/log.txt)

Are you using the recent version?

**Comment by MrSn0wy** [comment]
Yes, you can see the version in the title of the window (9af0047)

**Comment by GamingDucking** [comment]
Retest again

**Comment by JetPackAndButlers** [comment]
Boots, reaches the Splash screens and then crashes
With DISABLE_FMV_HACK it just hangs on a black screen instead of crashing
![immagine](https://github.com/red-prig/fpps4-game-compatibility/assets/91225364/13587145-dee8-4ec5-8a2f-b171655a5b3f)
![immagine](https://github.com/red-prig/fpps4-game-compatibility/assets/91225364/b0754746-815b-4fc6-8b87-ba294b8cc282)

**Comment by Jerome2b39** [comment]
![Capture d'écran 2024-10-24 171629](https://github.com/user-attachments/assets/ce712d61-8568-4a1d-a0dd-6404554b946c)

![Capture d'écran 2024-10-24 173658](https://github.com/user-attachments/assets/d1d7476f-7303-45ef-9c00-8127273d2ace)

I didn't put any hacks ^^

**Comment by gandalfthewhite19890404** [comment]
So, any information else?

**Comment by Jerome2b39** [comment]
I only have this unfortunately my friend =/
